templates/sectionsummary.html

```diff
@@ -51,3 +46,15 @@ <h2>{{summary.title}}</h2>
       {% endfor %}
   {% endif %}
 {% endblock %}
+
```

Comment:
Do we test this new button text on section summary anywhere?

Reply:
We do now